code/modules/mob/living/carbon/xenomorph/Castes/Ravager.dm

```diff
@@ -110,7 +110,7 @@
 /mob/living/carbon/Xenomorph/Ravager/ravenger/New()
 	..()
 	verbs -= /mob/living/carbon/Xenomorph/verb/hive_status
-	spawn(15) name = "Ravenger"
+	name = "Ravenger"
```

Comment:
No I was asking why this was spawned before. This looks like some fix that was added in cm because of some race condition bug. You shouldn't need this at all since the name is set in the type definition.